webapp/stores/emoji_store.jsx

```diff
@@ -148,8 +149,7 @@ class EmojiStore extends EventEmitter {
 
     getEmojiImageUrl(emoji) {
         if (emoji.id) {
-            // must match Client.getCustomEmojiImageUrl
-            return `/api/v3/emoji/${emoji.id}`;
+            return Client.getCustomEmojiImageUrl(emoji.id);
```

Comment:
I had a look at the issue and it says `react-router/es6`, which is excluded from the bundled script by `nodeExternals` uses `import` syntax, which is not supported by Node.js. Though you can whitelist it, that won't solve the issue because `react-router/es6` and other codes required by `web_client.jsx` requires a browser-like environment.